xradar/src/main/java/com/orzangleli/radar/XRadarView.java

```diff
@@ -374,9 +423,14 @@ private void drawPolygon(Canvas canvas) {
         }
     }
 
-    // 画 各层的颜色
-    @RequiresApi(api = Build.VERSION_CODES.KITKAT)
-    private void drawLayer(Canvas canvas, int startColor, int endColor) {
+    /**
+     * Draw the colors of each layer
+     *
+     * @param canvas     instance of Canvas
+     * @param startColor start color
+     * @param endColor   end color
+     */
+    private void drawLayer(Canvas canvas, Color startColor, Color endColor) {
         Path path = null;
```

Comment:
Null initialization is redundant here, there are similar issues at other places too, please fix them all.

Reply:
Remove null redundancy.